test/GithubApi.Redirect.test.js

```diff
@@ -0,0 +1,27 @@
+const axios = require('axios');
+const { expect } = require('chai');
+const { StatusCodes } = require('http-status-codes');
+
+const repository = {
+  url: 'https://github.com/aperdomob/redirect-test'
+};
+
+describe('Consume HEAD method and test redirecting url', () => {
+  it('Verify that response has redirected url with HEAD method', async () => {
+    const response = await axios.head(repository.url);
+
+    expect(response.status).to.equal(StatusCodes.OK);
+    expect(response.request.res.responseUrl).to.equal(
+      'https://github.com/aperdomob/new-redirect-test'
+    );
+
+    repository.redirectedUrl = response.request.res.responseUrl;
+  });
+
+  it('Verify that response has redirected url with GET method', async () => {
+    const response = await axios.get(repository.url);
+
+    expect(response.status).to.equal(StatusCodes.OK);
+    expect(response.request.res.responseUrl).to.equal(repository.redirectedUrl);
```

Comment:
There is a property of tests that they should be independent. So, this `it` should be able to execute without executing the first one.

If you want to use a variable to the expected value, better define it as a constant at the beggining, with something like
`const expectedRedirectUrl = 'https://github.com/aperdomob/new-redirect-test'`, and use it on both expects :D